libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/idp/IDPAccountPickerActivity.java

```diff
@@ -205,4 +216,32 @@ private void proceedWithIDPAuthFlow(UserAccount account) {
         intent.putExtra(IDPCodeGeneratorActivity.USER_ACCOUNT_BUNDLE_KEY, account.toBundle());
         startActivityForResult(intent, SPRequestHandler.IDP_REQUEST_CODE);
     }
+
+    /**
+     * A simple receiver that listens for IDP login completion. It fetches the user account
```

Comment:
This comments explains why we can't use `onActivityResult()` here. Hence, we use a `BroadcastReceiver` instead to receive results from `LoginActivity` (the account just added).